electric_waltz/scenario.py

```diff
@@ -198,11 +198,15 @@ def run(self) -> ScenarioRun:
         """
         num_steps = len(self._load)
 
-        power_sources = (
-            cast(list[PowerSource], self._baseloads)
-            + cast(list[PowerSource], [source for source, _ in self._intermittents])
-            + cast(list[PowerSource], self._flexibles)
```

Comment:
I think this should work in 3.8 if you replace `list` with `List` (imported from `typing`).

Reply:
Will look into this later.